backend/src/main/java/gov/cdc/usds/simplereport/config/DataHubClientConfiguration.java

```diff
@@ -29,7 +29,7 @@ public RequestInterceptor headerRequestInterceptor() {
       //
       // As a hack, set an empty request body to force an accurate Content-Length
       // header.
-      if (template.method() == HttpMethod.POST.toString() && template.body() == null) {
+      if (template.method().equals(HttpMethod.POST.toString()) && template.body() == null) {
```

Comment:
`==` checks for literal object equality, while `.equals` is overwritten for String to check that the string contents are identical. We want the latter in this case.

https://github.com/CDCgov/prime-simplereport/security/code-scanning/71